src/member_state_manager.ts

```diff
@@ -51,30 +51,24 @@ export class MemberState {
         return Date.now() - this.start_wait_timestamp
     }
 
-    TryAddToQueue(queue: HelpQueue, interaction_type: string): string | void {
+    TryAddToQueue(queue: HelpQueue, interaction_type: string): void {
         if(interaction_type === 'APPLICATION_COMMAND'){
             if(this.current_queue !== null) {
                 throw new UserError('Already enqueued')
             } else if (this.start_helping_timestamp !== null) {
                 throw new UserError('You can\'t join a queue while hosting')
             }
         }
-        //console.log('enqueuetest')
         if(this.current_queue !== null) {
-            //console.log('enqueuetest-case1')
-            return 'Already enqueued'
-            //throw new UserError('Already enqueued')
+            throw 'Already enqueued'
```

Comment:
Can we throw a UserError here as well? It seems to me that any of the UserErrors above should apply equally when using the buttons, so I don't see why these errors need to be different.

The difference should be how the error messages are delivered when using buttons vs. slash commands. If you plan on adding more buttons in the future, it might be worthwhile adapting the existing UserError handler to switch where these messages are delivered based on the interaction type.